functions/join.go

```diff
@@ -251,6 +251,7 @@ func JoinNetwork(flags *viper.Viper) (*config.Node, *config.Server, error) {
 		return nil, nil, errors.New("no network provided")
 	}
 	host := config.Netclient()
+	host.Name = flags.GetString("name")
```

Comment:
what's the behaviour if name is not passed?

Reply:
default value is set already using `os.Hostname()`

Reply:
Cool